drivers/ieee802154/Kconfig

```diff
@@ -139,7 +139,7 @@ config IEEE802154_2015
 
 config IEEE802154_DELAY_TRX_ACC
 	int "Clock accuracy for delayed operations"
-	default 20
+	default CLOCK_CONTROL_NRF_ACCURACY
```

Comment:
This is not possible since it's an NRF only symbol.

Reply:
The option IEEE802154_DELAY_TRX_ACC is used only by nrf driver. It is misplaced. It should be in nrf Kconfig.

Reply:
Because no other platform implements CSL yet, but should be possible to configure it.

Reply:
The option is used only in the nrf5 implementation of radio api (get_sch_acc specifically). By other platform it can be implemented in any way.